kin-sdk/kin-sdk-lib/src/main/java/kin/sdk/KinAccount.java

```diff
@@ -137,6 +156,66 @@
     @NonNull
     Balance getBalanceSync() throws OperationFailedException;
 
+    /**
+     * Create {@link Request} for getting the current confirmed aggregated balance in kin
+     * <p> See {@link KinAccount#getAggregatedBalanceSync()} for possibles errors</p>
+     *
+     * @return {@code Request<Balance>} Balance - the aggregated balance in kin
+     */
+    @NonNull
+    Request<Balance> getAggregatedBalance();
+
+    /**
+     * Get the current confirmed aggregated balance in kin
+     * <p><b>Note:</b> This method accesses the network, and should not be called on the android main thread.</p>
+     *
+     * @return the aggregated balance in kin
+     * @throws AccountNotFoundException if account was not created
+     * @throws OperationFailedException any other error
+     */
+    @NonNull
+    Balance getAggregatedBalanceSync() throws OperationFailedException;
+
+    /**
+     * Create {@link Request} for getting list of controlled accounts
+     * <p> See {@link KinAccount#getControlledAccountsSync()} for possibles errors</p>
+     *
+     * @return {@code Request<List<ControlledAccount>>} List<ControlledAccount> - the list of controlled accounts
+     */
+    @NonNull
+    Request<List<ControlledAccount>> getControlledAccounts();
+
+    /**
+     * Get the current list of controlled accounts
+     * <p><b>Note:</b> This method accesses the network, and should not be called on the android main thread.</p>
+     *
+     * @return the list of controlled accounts
+     * @throws AccountNotFoundException if account was not created
+     * @throws OperationFailedException any other error
+     */
+    @NonNull
+    List<ControlledAccount> getControlledAccountsSync() throws OperationFailedException;
+
+    /**
+     * Create {@link Request} for getting the current account data
+     * <p> See {@link KinAccount#getAccountDataSync()} for possibles errors</p>
+     *
+     * @return {@code Request<AccountData>>} AccountData - the current account data
+     */
+    @NonNull
+    Request<AccountData> getAccountData() throws OperationFailedException;
```

Comment:
we should really get rid of those multiple methods and have `Request` expose sync/async methods

Reply:
Will add it to our technical depth